src/format/opamFilter.ml

```diff
@@ -204,9 +207,9 @@ let resolve_ident ?(no_undef_expand=false) env fident =
      | Some (S s) -> FString s
      | None -> FUndef (FIdent fident))
 
-(* Resolves ["%{x}%"] string interpolations *)
-let expand_string ?(partial=false) ?default env text =
-  let default fident = match default, partial with
+(* Resolves ["%{x}%"] and ["%<x>%"] string interpolations *)
+let rec expand_string_aux ?(regex=path_interp_regex) ?(partial=false) ?default env text =
```

Comment:
Note that the use of `~regex` means that this only introduces one recursive call - expansion of `%{...}%` within `%<...>%`. It **doesn't** add recursive expansion of variables by accident!